php_phongo.c

```diff
@@ -1870,7 +1870,23 @@ mongoc_client_t *php_phongo_make_mongo_client(const mongoc_uri_t *uri, zval *dri
 		php_phongo_populate_default_ssl_ctx(ctx, driverOptions);
 	}
 
-	MONGOC_DEBUG("Creating Manager, phongo-%s[%s] - mongoc-%s, libbson-%s", MONGODB_VERSION_S, MONGODB_STABILITY_S, MONGOC_VERSION_S, BSON_VERSION_S);
+	MONGOC_DEBUG("Creating Manager, phongo-%s[%s] - mongoc-%s(%s), libbson-%s(%s), php-%s",
+		MONGODB_VERSION_S,
+		MONGODB_STABILITY_S,
+		MONGOC_VERSION_S,
+#ifdef HAVE_SYSTEM_LIBMONGOC
```

Comment:
This appears to trigger [Compiler Error C2121](https://msdn.microsoft.com/en-us/library/1s467t6a.aspx) in VC11, because we're using a macro within the `MONGOC_DEBUG()` macro. I believe a quick solution would be to populate `const char *` variables beforehand and use those as-is in the `MONGOC_DEBUG()` macro.